src/ResourceManager/AzureBatch/ChangeLog.md

```diff
@@ -22,6 +22,8 @@
 ## Version 4.0.7
 * Set minimum dependency of module to PowerShell 5.0
 * Updated New-AzureBatchPool documentation to remove deprecated example
+* Release new cmdlet Get-AzureBatchPoolNodeCounts
```

Comment:
@antoniowmsft please move the snippet that you added to below the `## Current Release` header

Reply:
That's why I asked earlier if I need to edit changelog.md. 👍